src/config/graphical-session/kde.md

```diff
@@ -0,0 +1,17 @@
+# KDE
+
+## Installation
+
+Install the `kde5` package, and optionally, the `kde5-baseapps` package.
+
+To use the "Networks" widget, enable the `dbus` and `NetworkManager` services.
+
+Installing the `kde5` package also installs the `sddm` package, which provides
+the `sddm` service for the Simple Desktop Display Manager; [test the
+service](../services/managing.html#testing-services) before enabling it. If you
+are not intending to run SDDM via a remote X server, you will need to install
+either the `xorg-minimal` package or the `xorg` package. SDDM defaults to
+providing an Xorg session, but a Wayland session is available.
+
+If you wish to start an Xorg-based KDE session from the console, use
+[startx](./xorg.html#startx) to run `startplasma-x11`.
```

Comment:
If you want to be thorough, you can mention that there's a `startplasma-wayland` program for the Wayland session.

Reply:
It wouldn't be launched through `startx`, of course.

Reply:
@ericonr: Good suggestion; i'll add that in.